OpenFGA Project security self-assessment update (#1235)
cncf/tag-security: assessments/projects/openfga/self-assessment.md

# OpenFGA Security Self Assessment

> This assessment was initially drafted by the core maintainer team and contributed to by community members as part of the [Security Pals](https://github.com/cncf/tag-security/issues/1102) process. 

Reviewer: Maybe this?
```suggestion
This assessment was contributed to by community members as part of the [Security Pals](https://github.com/cncf/tag-security/issues/1102) process as well as the core maintainer team. 

```

# OpenFGA Security Self Assessment

> This assessment was created by community members as part of the [Security Pals](https://github.com/cncf/tag-security/issues/1102) process and is currently pending changes from the maintainer team.

Reviewer 2: Would it be okay to change this so it acknowledges the Security Pals effort, but indicates the maintainers have completed the process?

Author: Totally.

>   configuration.language:

Reviewer: ```suggestion
  openfga.language:

```

    - Provide a domain specific language to describe authorization policies

>     - Describe the authorization model with[types], [relations] and [conditions] 

Reviewer: ```suggestion
    - Describe the authorization model with [types], [relations] and [conditions] 

```
